### PR TITLE
State scripts timeout

### PR DESCRIPTION
**Pull request by kacf**
```
commit c5b3e066824a10b9998e8fbf4393b538a13648b3
Author: Kristian Amlie <kristian.amlie@northern.tech>
Date:   Thu Sep 7 13:58:21 2017

    Link state scripts timeout paragraph to configuraion options list.
    
    Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

commit 8e85b267bfc18458f2228e54a0f2ee5864889310
Author: Kristian Amlie <kristian.amlie@northern.tech>
Date:   Thu Sep 7 13:43:33 2017

    Document using a custom mender.conf file.
    
    This is preferred over Yocto variables for most non-critical settings,
    in order to avoid polluting the build with a lot of global variables
    for small details.
    
    Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>
```

### CODE REVIEW

File: 05.Client-configuration/04.Configuration-file/docs.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Configuration file
+taxonomy:
+    category: docs
+---
+
+A lot of the Mender client's configuration resides in `/etc/mender/mender.conf`
```
Comment (drewmoseley) [inline 05.Client-configuration/04.Configuration-file/docs.md]: Much of the Mender client's configuration ...

File: 05.Client-configuration/04.Configuration-file/docs.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+on the root filesystem. This file is a JSON structure that defines various
```
Comment (drewmoseley) [inline 05.Client-configuration/04.Configuration-file/docs.md]: This file is JSON structured and defines...

File: 05.Client-configuration/04.Configuration-file/docs.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+parameters for Mender's operation. Some of the most common settings are tunable
+using [Yocto
+variables](../artifacts/image-configuration#configuring-polling-intervals), but
```
Comment (drewmoseley) [inline 05.Client-configuration/04.Configuration-file/docs.md]: variables.  The remaining parameters can only be changed by providing your own mender.conf.

File: 05.Client-configuration/04.Configuration-file/docs.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+recipe. It should be placed in a location like this:
+`meta-<mylayer>/recipes-mender/mender/mender_%.bbappend`. The content of the
+file should be the following:
+
+```
+FILESEXTRAPATHS_prepend = "${THISDIR}/files:"
+SRC_URI_append = " file://mender.conf"
+```
+
+If the `mender_%.bbappend` file already exists for other reasons, it is alright
```
Comment (drewmoseley) [inline 05.Client-configuration/04.Configuration-file/docs.md]: This reads a bit awkward.  I suggest instead, "If your layer already contains a mender_%.bbappend file, then add the above contents into your file, taking care to not redefine any variables you are already using."

**Comment by kacf** [comment]
@drewmoseley: Fixed all!

**Comment by drewmoseley** [comment]
Looks great.  Merging.

**Pull request merged**